src/App.js

```diff
@@ -1,15 +1,11 @@
+import Layout from "./layouts/Layout";
 import "./App.css";
-import Header from "./components/Header";
-import Content from "./components/Content";
-import Footer from "./components/Footer";
 
 function App() {
   return (
```

Comment:
worth of reading, declaring component by `function`` or arrow function:
https://stackoverflow.com/questions/54331084/function-or-fat-arrow-for-a-react-functional-component

I'm using arrow functions to declare components and most projects, where I worked was doing the same.

Reply:
Changed to arrow functions.